.github/workflows/build-test-tidy-pr.yaml

```diff
@@ -50,88 +50,30 @@ jobs:
 
   build-and-test-differential-cuda:
     needs: check-if-cuda-job-is-needed
-    if: ${{ needs.check-if-cuda-job-is-needed.outputs.cuda_job_is_needed == 'true' }}
     uses: ./.github/workflows/build-and-test-differential.yaml
     with:
       container: ghcr.io/autowarefoundation/autoware:universe-devel
       container-suffix: -cuda
+      run-condition: ${{ needs.check-if-cuda-job-is-needed.outputs.cuda_job_is_needed == 'true' }}
     secrets:
       codecov-token: ${{ secrets.CODECOV_TOKEN }}
 
-  build-test-pr:
-    needs:
-      - build-and-test-differential
-      - build-and-test-differential-cuda
-    if: ${{ always() }}
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Initialize Summary
-        run: echo "### Build Test PR Results" > $GITHUB_STEP_SUMMARY
-        shell: bash
-
-      - name: Evaluate build-and-test-differential
-        uses: ./.github/actions/evaluate-job-result
-        with:
-          job_result: ${{ needs.build-and-test-differential.result }}
-          job_name: build-and-test-differential
-          expected_results: success
-
-      - name: Evaluate build-and-test-differential-cuda
-        if: ${{ always() }}
-        uses: ./.github/actions/evaluate-job-result
-        with:
-          job_result: ${{ needs.build-and-test-differential-cuda.result }}
-          job_name: build-and-test-differential-cuda
-          expected_results: success,skipped
-
   clang-tidy-differential:
+    if: ${{ always() }} # always run to provide report for status check
     needs:
       - check-if-cuda-job-is-needed
       - build-and-test-differential
-    if: ${{ needs.check-if-cuda-job-is-needed.outputs.cuda_job_is_needed == 'false' }}
     uses: ./.github/workflows/clang-tidy-differential.yaml
     with:
       container: ghcr.io/autowarefoundation/autoware:universe-devel
+      run-condition: ${{ needs.check-if-cuda-job-is-needed.outputs.cuda_job_is_needed == 'false' }}
 
   clang-tidy-differential-cuda:
+    if: ${{ always() }} # always run to provide report for status check
     needs:
       - build-and-test-differential-cuda
```

Comment:
I had to explicitly set "always()" condition to make sure that "clang-tidy-differential-cuda" job even when build-and-test-differential-cuda is not ran.